src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts

```diff
@@ -920,25 +920,29 @@ export interface Osmosis1TrxMsgOsmosisIncentivesMsgAddToGauge
 }
 
 // types for mgs type:: /osmosis.incentives.MsgCreateGauge
-export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge
-  extends IRangeMessage {
-  type: Osmosis1TrxMsgTypes.OsmosisIncentivesMsgCreateGauge;
-  data: {
-    coins: {
-      denom: string;
-      amount: string;
-    }[];
+export interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGauge {
+    type: string;
+    data: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData;
+}
+interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeData {
     owner: string;
+    distributeTo: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo;
+    coins: Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeCoinsItem[];
     startTime: string;
-    distributeTo: {
-      denom: string;
-      duration: string;
-      timestamp: string;
-    };
     numEpochsPaidOver: string;
-  };
+    poolId: string;
+}
+interface Osmosis1TrxMsgOsmosisIncentivesMsgCreateGaugeDistributeTo {
+    lockQueryType: string;
```

Comment:
`denom` is an optional string here

Reply:
This is the proto:
message QueryCondition {
  LockQueryType lock_query_type = 1;
  // Denom represents the token denomination we are looking to lock up
  string denom = 2;
  google.protobuf.Duration duration = 3;
  google.protobuf.Timestamp timestamp = 4;
}